RizeFinance/prototype-ui
# Rf 908 virtual card

[Whimsical Specs](https://whimsical.com/prototype-core-B152vwoVF6RpZSGqxGsimH)

Testing Tips:

1. Create a new user via "I need to create a new account"
2. Check out Debit Cards. 
3. Lock / Unlock it
4. Report stolen
5. Migrate to Physical
6. Lock / Unlock it
7. Report stolen (hasn't been updating with a newly issued card lately...)

Note: Pin set is in mock mode and will not actually set a pin
